tool/inits/GEN_UM_IC_WITH_PHASE_WITH_TRANSFER_FUNC/set_velocities_zero.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3.7
+
+##################################################################################################
+# This script is used for setting the velocity-dependent transfer fucntion (TF) to 1e-8 for TF   #
+# file generated by CAMB(planck_2018_transfer_out.dat) and axsionCAMB(planck_2018_axion_transfer #
+# _out.dat). Be sure to prepare the input transfer function files and specify their name!! The   #
+# output file planck_2018_transfer_out_no_vel.dat/planck_2018_axion_transfer_out.dat will be     #
+# produced accordingly(corresponding to CAMB/axionCAMB input).                                   #
+##################################################################################################
+
+import sys
+import numpy as np
+import pandas as pd
+
+if len(sys.argv) != 2:
+    raise RuntimeError("Input Error: Please select do axion or not (T/F)!")
```

Comment:
Somehow I don't feel intuitive to pass the argument `T` for axion and `F` for not axion.

Reply:
I have changed `set_velocities_zero.py` as below:
```
if len(sys.argv) != 3:
    raise RuntimeError("Input Error: Please select whether-axion-physics-included (T/F) and file name of input transfer function!")
else:
    if sys.argv[1] not in ["T","F"]:
        raise RuntimeError("Input Error: Please select T/F if the input transfer function includes/excludes axion-physics!")
    input_file = sys.argv[2]
```
, and modified the `README.txt` as:
```
(b) Run the script: "python3 ./set_velocities_zero.py F(T) planck_2018_trnasfer_out.dat(planck_2018_axion_trnasfer_out.dat)".
    The script reads-in the output files in step(a), and produce the transfer function requireded by MUSIC.
    The boolean F/T suggests whether axion-physics is included in the input transfer function files.
```